test/WebJobs.Script.Tests.Integration/Controllers/ControllerScenarioTestFixture.cs

```diff
@@ -29,16 +35,22 @@ public ControllerScenarioTestFixture(bool isAuthDisabled)
             HostSettings = new WebHostSettings
             {
                 IsSelfHost = true,
-                ScriptPath = Path.Combine(Environment.CurrentDirectory, @"..\..\..\..\sample"),
+                ScriptPath = Path.Combine(Environment.CurrentDirectory, @"..\..\..\..\..\sample"),
```

Comment:
For the below SecretsPath - where is this path in this repo? There isn't an App_Data folder anymore

Reply:
Good point! This was not really relevant here, so I was going to use a temp folder missed that. I'll have that updated.

Reply:
Updated